### ISSUE TITLE
css class not found

### ISSUE THREAD

**Issue by matardy** [joeyism/linkedin_scraper#128]
```
 self.name = driver.find_element_by_class_name(
            "top-card-layout__title"
        ).text.strip()
```

Raise this error: 

`selenium.common.exceptions.NoSuchElementException: Message: no such element: Unable to locate element: {"method":"css selector","selector":".top-card-layout__title"}`

Any updates on linkedin 2022?

**Comment by ernests** [comment]
It looks like Linkedin has made UI changes and by doing so also changed class names. As this tool is scraping UI it can't locate stuff anymore.

**Comment by ianbongs** [comment]
wait so does this mean this app can't work at all now?

**Comment by matardy** [comment]
Yes, it is actually a challenge to make scraping from linkedin because its UI keeps in constant change.

**Comment by valentijnverberk** [comment]
Facing the same issue - how can we get it to work?

**Comment by matardy** [comment]
We need to study LinkedIn Web Structure and change all css classes names. 
For example the container who shows the work experience in a LinkedIn profile used to have the name "experiences" now the name is "css-selector-8..." and so.

**Comment by 1zuu** [comment]
What @matardy says is true.  we can't use this anymore unless they update the tool for LinkedIn 2022 UI version
